### PR TITLE
Add swedish translation for police_horse

### PR DESCRIPTION
**Pull request by itayo**

### CODE REVIEW

**Review by Cr4zyc4k3 — DISMISSED** [review]

**Comment by Cr4zyc4k3** [comment]
Looks good for me. This modules also need translations :see_no_evil: 
- creditsextension
- dailyCreditsSummary
- extendedCallWindow
- extendedMap
- heatmap
- hotkeys
- messageTemplates
- redesign
- statusCounter

https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/creditsextension
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/dailyCreditsSummary/
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/extendedCallWindow/
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/extendedMap/
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/heatmap/
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/hotkeys/
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/messageTemplates
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/redesign/
https://github.com/LSS-Manager/LSSM-V.4/tree/dev/src/modules/statusCounter/

**Comment by itayo** [comment]
> * creditsextension
> * dailyCreditsSummary
> * extendedCallWindow
> * extendedMap
> * heatmap
> * hotkeys
> * messageTemplates
> * redesign
> * statusCounter

Il take alook at them

**Comment by Cr4zyc4k3** [comment]
Thanks in advance. Don't feel liable to do all translations.It was just a quick idea.
You can add them in this MR.

**Comment by itayo** [comment]
[ ] creditsextension
[ ] dailyCreditsSummary
[ ] extendedCallWindow
[ ] extendedMap
[ ] heatmap
[ ] hotkeys
[ ] messageTemplates
[ ] redesign
[ ] statusCounter

**Comment by jxn-30** [comment]
> As im guessing you guys have to verify langauge its better i do 1 MR per module imho
It is also ok to have everything in one MR, which would also be a little less work. So do it the way, you're more comfortable with.
Also, we're not checking language, as we don't speek most of the languages ^^

File: src/modules/creditsextension/i18n/sv_SE.root.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"nextRankMissing": "kvarvarande",
+	"settings": {
+		"alerts": {
+			"credits": "krediter",
+      "description": "När du nått en av de fördefinerade kreditnivåerna kommer kredit expansionen markeras som grön tills du har tryckt på demn.",
+			"title": "Mål"
+		},
+		"coin​
+2
+sInNavbar": {
```
Comment (jxn-30) [inline src/modules/creditsextension/i18n/sv_SE.root.json:27]: there accidentally is a `2` within the text ;)

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**